hadoop-tools/hadoop-aws/src/main/java/org/apache/hadoop/fs/s3a/Constants.java

```diff
@@ -1390,6 +1390,13 @@ private Constants() {
    */
   public static final String FS_S3A_CREATE_PERFORMANCE = "fs.s3a.create.performance";
 
+  /**
+   * Flag for commit if none match.
+   * This can be set in the {code createFile()} builder.
+   * Value {@value}.
+   */
+  public static final String FS_S3A_CREATE_IF_NONE_MATCH = "fs.s3a.create.header.If-None-Match";
```

Comment:
This property name looks out of pattern, it doesn't look like we use upper case letters elsewhere. I think the correct name would be `fs.s3a.create.header.if.none.match`

Reply:
+1, should change to `fs.s3a.create.header.if.none.match`

Reply:
updated